autoscheduler/scraper/models/course.py

```diff
@@ -1,4 +1,18 @@
 from django.db import models
 
-class Course(models.Model):
-    """ Describe model here """
+class Course(models.Model): # Directly from GBS:w
+    id = models.CharField(max_length=10, primary_key=True) # Could be dept+course_num
+    dept = models.CharField(max_length=4, db_index=True) # CSCE
+    course_num = models.CharField(max_length=5, db_index=True) # i.e. 221
+    title = models.CharField(max_length=100) # Course title, i.e. "Data Structres & Algorithms"
+    description = models.TextField(null=True, blank=True)
+    prerequisites = models.TextField(null=True, blank=True)
+    corequisites = models.TextField(null=True, blank=True)
+    cd = models.CharField(max_length=3)#yes or no. states whether course is a cd credit
+    icd = models.CharField(max_length=3)#yes or no. states whether course is an icd credit
+    core_curriculum = models.CharField(max_length=32)#tag specifying what it counts as ex) 'Creative Arts' or 'American History'. If not a core curriculum class it can be blank or 'NA' or something

```

Comment:
> If not a core curriculum class it can be blank or 'NA' or something

Since it's possible to be blank, this should have `null=True` to indicate that the field can be empty

Reply:
I wasn't sure if you wanted to go with 'NA' or blank or something else in the case of a class not being part of the core curriculum, so I just left it as default with max length being the length of the longest classification. I shall add the null = true now that I know which one you want to go with.